tests/components/songpal/test_media_player.py

```diff
@@ -12,6 +12,7 @@
     SongpalException,
     VolumeChange,
```

Comment:
```suggestion
    VolumeChange,
    SettingChange,
```
Re my other comment below, shouldn't this work?

Reply:
Yes yes, you are right, I failed in my demonstration, because it was about Setting and not SettingChange.
I already have changed the SettingChange to import it in the songpal in the last commit, only 
```python
from songpal.containers import Setting
```
remains, but this one is not exported.

Reply:
Yeah, maybe the upstream lib should export that and maybe more, but we shouldn't block this PR for that. I'm fine with this PR as it is, so we are just waiting for @emontnemery to approve.

If you want to improve this meanwhile, you could add tests for those error cases that are causing the drop in coverage.